shub/image/deploy.py

```diff
@@ -55,16 +55,16 @@
 @click.option("--email", help="docker registry email")
 @click.option("--apikey", help="SH apikey to use built-in registry")
 @click.option("--insecure", is_flag=True, help="use insecure registry")
-@click.option("--async", is_flag=True, help="[DEPRECATED] enable asynchronous mode",
+@click.option("--async", "_async", is_flag=True, help="[DEPRECATED] enable asynchronous mode",
```

Comment:
I think the norm for using reserved keywords as variables names that are not private is to suffix (rather than prefix) with underscore.

Reply:
I don't like to include underscore as well. I prefixed with underscore to follow the example in Click
http://click.pocoo.org/dev/options/#basic-value-options
```
# How to use a Python reserved word such as `from` as a parameter
@click.command()
@click.option('--from', '-f', '_from')
@click.option('--to', '-t')
def reserved_param_name(_from, to):
    click.echo('from %s to %s' % (_from, to))
```

Reply:
that is an interesting conflict of standards! [PEP8 mentions](https://www.python.org/dev/peps/pep-0008/#descriptive-naming-styles) this about the topic:

> the following special forms using leading or trailing underscores are recognized (these can generally be combined with any case convention):
>
>   *  _single_leading_underscore: weak "internal use" indicator. E.g. from M import * does not import objects whose name starts with an underscore.
>
 >  * single_trailing_underscore_: used by convention to avoid conflicts with Python keyword, e.g.
>
>        Tkinter.Toplevel(master, class_='ClassName')

```